lib/hls/hls_parser.js

```diff
@@ -308,7 +308,13 @@ shaka.hls.HlsParser = class {
       const PresentationType = shaka.hls.HlsParser.PresentationType_;
       this.setPresentationType_(PresentationType.VOD);
 
-      const maxTimestamps = streamInfos.map((s) => s.maxTimestamp);
+      // Only check active stream:
```

Comment:
I am pretty sure this change is correct. It's actually a change I had been considering, while diagnosing an issue that wasn't filed on GitHub.

However, this block of comments is hard to parse, grammatically. And honestly, we probably don't need to go into this much detail.
I'd rephrase like this:
```js
      // The duration is the minimum of the end times of all active streams.
      // Non-active streams are not guaranteed to have useful maxTimestamp
      // values, due to the lazy-loading system, so they are ignored.
```

Reply:
Thanks.